src/cargo/core/workspace.rs

```diff
@@ -657,10 +657,15 @@ impl<'gctx> Workspace<'gctx> {
         self.honor_rust_version
     }
 
+    pub fn set_resolve_honors_rust_version(&mut self, honor_rust_version: Option<bool>) {
+        self.resolve_honors_rust_version = honor_rust_version;
+    }
+
     pub fn resolve_honors_rust_version(&self) -> bool {
```

Comment:
I am now seeing `resolve_honors_rust_version` and `honors_rust_version` quiet confusing. Any change to explain or conslidate them?

Reply:
BTW, not a blocker